### PR TITLE
Several crash and bug fixes

### PR DESCRIPTION
**Pull request by tylerblurie**

### CODE REVIEW

File: toontown/spellbook/MagicWordIndex.py [hunk 1]
```diff
@@ -316,7 +316,20 @@ def handleWord(self, invoker, avId, toon, *args):
         toon.b_setTickets(RaceGlobals.MaxTickets)
         maxTrophies = RaceGlobals.NumTrophies + RaceGlobals.NumCups
         toon.b_setKartingTrophies(range(1, maxTrophies + 1))
-        toon.b_setTickets(99999)
```
Comment (DarthMDev) [inline toontown/spellbook/MagicWordIndex.py:319]: Why is this removed ?

File: toontown/estate/DistributedPhoneAI.py [hunk 2]
```diff
@@ -225,7 +225,7 @@ def handleAccount(dclass, fields):
             if not item.getDeliveryTime():
                 retCode = item.recordPurchase(av, optional)
                 deliveryTime = 0
-            elif config.GetBool('want-instant-delivery', False):
+            elif av.instantDelivery == True:
```
Comment (DarthMDev) [inline toontown/estate/DistributedPhoneAI.py:228]: The config works fine why change it ?

Comment (DarthMDev) [reply toontown/estate/DistributedPhoneAI.py:228]: Besides it being deprecated (then use ConfigVariableBool)

Comment (tylerblurie) [reply toontown/estate/DistributedPhoneAI.py:228]: This was changed so that the ~InstaDelivery magic word would function correctly. Without it, invoking ~InstaDelivery does not cause items the invoker orders *after* using the magic word to be instantly delivered, The magic word is currently not programmed to change a config variable, and there currently is no ```want-instant-delivery``` variable in this source. Doing it this way allows for items to be instantly delivered on a per-invoker basis (like with ~SetAlwaysHitCogs and ~Immortal), rather than affecting every player on the server.

File: toontown/spellbook/MagicWordIndex.py (hunk 1 above)
Comment (tylerblurie) [reply toontown/spellbook/MagicWordIndex.py:319]: It is not necessary to have it twice. The max tickets are already set three lines above it on line 316:

```toon.b_setTickets(RaceGlobals.MaxTickets)```

**Review by DarthMDev — APPROVED** [review]
